tests/test_posts.py

```
@@ -0,0 +1,211 @@
+        data = response.json()
+        assert data["id"] == "post_00000001"
+        assert data["title"] == "첫 번째 글"
+        assert data["content"] == "내용1"
+
+    def test_get_single_post_view_count_increment(self, client, mock_posts_data):
+        """조회 시 view_count 증가 확인"""
+        original_view_count = mock_posts_data[0]["view_count"]
+
+        with patch("routers.posts.read_json", return_value=mock_posts_data):
+            with patch("routers.posts.write_json") as mock_write:
+                response = client.get("/posts/post_00000001")
```

Ruff (0.14.13)

Local variable `response` is assigned to but never used

Remove assignment to unused variable `response`

(F841)